tests/test_common/matrix.c

```diff
@@ -23,6 +23,7 @@ static matrix_row_t matrix[MATRIX_ROWS] = {};
 void matrix_init(void) {
     clear_all_keys();
     matrix_init_quantum();
+    matrix_init_user();
```

Comment:
This should be removed, as it should be called anyways.  Adding it here means that it will be called twice.
```suggestion
```

Reply:
Done, thanks!